### PR TITLE
Update Kafka/Kafka-connect scripts

### PR DESCRIPTION
**Pull request by kyguy**
(3) Third PR unit to review. Update scripts to what was in upstream/master

Issue: https://github.com/jboss-container-images/jboss-amqstreams-1-openshift-image/issues/1
Issue: https://github.com/jboss-container-images/jboss-amqstreams-1-openshift-image/issues/4

### CODE REVIEW

File: kafka-connect/modules/kafka-connect/added/scripts/kafka_connect_config_generator.sh [hunk 1]
```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
```
Comment (ppatierno) [inline kafka-connect/modules/kafka-connect/added/scripts/kafka_connect_config_generator.sh]: This file is changed one more time. Get the updated version from `master` branch of the `strimzi-kafka-operator` project.

File: templates/kafka-connect/kafka-connect-s2i.yaml [hunk 2]
```diff
@@ -0,0 +1,15 @@
+apiVersion: kafka.strimzi.io/v1alpha1
```
Comment (ppatierno) [inline templates/kafka-connect/kafka-connect-s2i.yaml]: This file is changed one more time. Get the updated version from `master` branch of the `strimzi-kafka-operator` project.
But I have a question on this one. It's not an OpenShift template, why it's under `templates` folder?

File: templates/kafka-connect/kafka-connect.yaml [hunk 3]
```diff
@@ -0,0 +1,15 @@
+apiVersion: kafka.strimzi.io/v1alpha1
```
Comment (ppatierno) [inline templates/kafka-connect/kafka-connect.yaml]: This file is changed one more time. Get the updated version from `master` branch of the `strimzi-kafka-operator` project.
Same question as above, it's not a template btw.

**Comment by kyguy** [comment]
Changes backported, not expecting any more upstream changes for this release

File: kafka-connect/modules/kafka-connect/added/scripts/kafka_connect_tls_prepare_certificates.sh [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+do
+    create_truststore /tmp/kafka/cluster.truststore.p12 $CERTS_STORE_PASSWORD /opt/kafka/connect-certs/$cert $cert
+done
+echo "Preparing truststore is complete"
+
+
+if [ -n "$KAFKA_CONNECT_TLS_AUTH_CERT" ] && [ -n "$KAFKA_CONNECT_TLS_AUTH_KEY" ]; then
+    echo "Preparing keystore"
+    create_keystore /tmp/kafka/cluster.keystore.p12 $CERTS_STORE_PASSWORD /opt/kafka/connect-certs/${KAFKA_CONNECT_TLS_AUTH_CERT} /opt/kafka/connect-certs/${KAFKA_CONNECT_TLS_AUTH_KEY} ${KAFKA_CONNECT_TLS_AUTH_CERT}
+    echo "Preparing keystore is complete"
```
Comment (ppatierno) [inline kafka-connect/modules/kafka-connect/added/scripts/kafka_connect_tls_prepare_certificates.sh]: Is there an "fi" missing for closing the if statement?

Comment (kyguy) [reply kafka-connect/modules/kafka-connect/added/scripts/kafka_connect_tls_prepare_certificates.sh]: fixed

**Pull request merged**